h/admin/services/user.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
This seems like a slightly odd place to put this to me. It can stay here for now, but in general I think services should live alongside the models that they are (principally) involved with manipulating, i.e. in `h.accounts.services` in this case.